apps/molgenis-components/src/components/layout/Molgenis.vue

```diff
@@ -28,22 +28,25 @@
         <slot />
       </div>
     </div>
-    <MolgenisFooter>
-      <span v-if="session?.manifest">
-        Software version:
-        <a
-          :href="
-            'https://github.com/molgenis/molgenis-emx2/releases/tag/v' +
-            session.manifest.SpecificationVersion
-          "
-        >
-          {{ session.manifest.SpecificationVersion }} </a
-        >.
-        <span v-if="session.manifest.DatabaseVersion">
-          Database version: {{ session.manifest.DatabaseVersion }}.
+    <div>
```

Comment:
I think it would be good to wrap the footer element in the footer tag to ensure semantic HTML. We may want to consider changing the `div` to the `footer` component. Ideally, the main slot should be wrapped in the `main` component, but we can address that when we redesign the component library

Reply:
I also see that the `MolgenisFooter` component does not contain the footer element.